hypha/apply/funds/templates/funds/includes/screening_form.html

```diff
@@ -1,6 +1,6 @@
 {% if screening_form.should_show %}
 <div class="modal" id="screen-application" data-yes-statuses="{{ submission.yes_screening_statuses }}" data-no-statuses="{{ submission.no_screening_statuses }}" data-default-yes="{{ default_yes }}" data-default-no="{{ default_no }}">
-    <h4 class="modal__header-bar">Update status</h4>
+    <h4 class="modal__header-bar">Screen application</h4>
```

Comment:
Unrelated change but noticed the header was wrong so opted for adding a fix here.